### PR TITLE
CLN: assorted

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/core/ops/__init__.py [hunk 1]
```diff
@@ -312,7 +315,7 @@ def to_series(right):
 
 
 def should_reindex_frame_op(
-    left: DataFrame, right, op, axis, default_axis, fill_value, level
+    left: DataFrame, right, op, axis: int, fill_value, level
```
Comment (WillAyd) [inline pandas/core/ops/__init__.py:318]: Not a big deal but I guess we have AxisInt in pandas._typing for this already

**Review by WillAyd — COMMENTED** [review]
probably would have been better to break out some of the actual code / test changes but generally think this is OK

File: pandas/core/ops/__init__.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/core/ops/__init__.py:318]: will update in next pass, dont want to futher clog the CI

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
LGTM merge when ready @WillAyd

**Comment by WillAyd** [comment]
Thanks @jbrockmendel

**Pull request merged**